pdr_backend/cli/cli_arguments.py

```diff
@@ -545,6 +548,7 @@ def network_choices(self):
     "do_multisim": MultisimArgParser("Run >1 simulations", "multisim"),
     "do_deployer": DeployerArgPaser(),
     "do_lake": LakeArgParser("Run the lake tool", "lake"),
+    "do_lake_describe": LakeArgParser("Show lake infol", "lake_describe"),
```

Comment:
Typo. 'infol' -> 'info'

Reply:
This looks resolved in the LakeArgParser.